src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String text, String filePath) {
+        try (BufferedWriter bw = new BufferedWriter(new java.io.FileWriter(filePath))) {
```

Comment:
Consider using try-with-resources with java.io.FileWriter to ensure proper resource management. This is already done, but the explicit import for FileWriter is missing.